tests/Routing/RoutingRouteTest.php

```diff
@@ -546,6 +546,16 @@ public function testModelBindingWithBindingClosure()
         $this->assertEquals('tayloralt', $router->dispatch(Request::create('foo/TAYLOR', 'GET'))->getContent());
     }
 
+    public function testModelBindingThroughIOC()
+    {
+        $router = new Router(new Illuminate\Events\Dispatcher, $container = new Illuminate\Container\Container);
```

Comment:
Probably need to go through whole test class at some point because it's not using an import at all for `Dispatcher` or `Container`. Added it anyway